.storybook/preview.js

```diff
@@ -102,36 +99,46 @@ export const globalTypes = {
   locale: {
     name: "Locale",
     description: "Internationalization locale",
-    defaultValue: "en_US",
+    defaultValue: "en-US",
     toolbar: {
       icon: "globe",
       items: [
-        { value: "en_US", right: "🇺🇸", title: "English" },
-        { value: "pseudo", right: "🤪", title: "Pseudo localization" },
+        { value: "en-US", right: "🇺🇸", title: "English (US)" },
+        { value: "en-GB", right: "🇬🇧", title: "English (GB)" },
```

Comment:
So are we differentiating between US and GB english now? We'll need to find a translator for the GB English ;-P

Reply:
Actually no! We only use the main part of the language (so, en) but this is handy to check that this mechanism works since browsers will give us the full language plus variant (so, en-GB)

Reply:
Italian too, it is just to check that the fallback to en works.